### ISSUE TITLE
Formating causes wrong indent

### ISSUE THREAD

**Issue by sseypt** [prettier/plugin-php#390]
Hello.

Input:
```php
<?php
if ($parity != array_reduce($r,function ($acc, $tmp) {return $acc ^ $tmp;},0)) {
    return [];
}

```

Output:
```php
<?php
if ($parity != array_reduce(
        $r,
        function ($acc, $tmp) {
            return $acc ^ $tmp;
        },
        0
    )) {
    return [];
}

```

Expected:
```php
<?php
if ($parity != array_reduce(
    $r,
    function ($acc, $tmp) {
        return $acc ^ $tmp;
    },
    0
)) {
    return [];
}

```

Best regards,
Steve

**Comment by sseypt** [comment]
Another example:

Input:
```php
<?php
implode(
    ', ',
    array_map(
        function ($f) {
            return $f;
        },
        array_merge($arr1, $arr2, $arr3)
    )
);

```

Output:
```php
<?php
implode(', ', array_map(function ($f) {
    return $f;
}, array_merge($arr1, $arr2, $arr3)));

```

Expected:
```php
<?php
implode(
    ', ',
    array_map(function ($f) {
        return $f;
    }, array_merge($arr1, $arr2, $arr3))
);

```

**Comment by evilebottnawi** [comment]
1. Should be https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAZgAgBQBIAOAhgE6owCemAhALyYnGHkD6xcAJgK5hzYA6UTEOFDcxADQCRI9JyhgYqaDlyEwYcZlwwAtvgCUmYFOmm2MTsUGr1mAHpbd+ANwnTAX0mDTmAAxvMfUNjbyFzS0EAbQBdVyh3EHEQCHxFaABnZFAGCAB3AAUSBEyUQgA3CFR2RJBCdJhkdEIAG3S4JIAjRjAAazgYAGUiMFQoAHNkGGJOdpB2CDBGlrak0bbiGHzGMZ1CJdbZgCt0gA8AIW6+wcIdOAAZUbh9leTOGHw3gCZn2aJideQIA6hA65Ga0Bq+FIsAA6lUYAALZAADl8SShEDaMMY+EBULg6zKTySbAAjpxUGwtoQdnskE0Dkk2jpUJNprN0qMxs04ABFTgQeA-JIwEFw9iI5CfEWMVDNLkAYQgOl2gKg0GJIE4bQAKiCSgy2u53EA
2. Should be https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAtgBwDYQCZwAUAOlAAQVkDkANNTaZWQIYBOrzAngPrrOYlyTSgDMArlDAxU0MoQAkIgJRlgjYRtZwYY1uUUBudRoC+DIRrYce6OKwDmReVYCMdZ+wBM7qwGYlxmQBUEoGIDQgEJjS0ADOyKBWEADuAApsCPEozABuEKh44SDMsTDIIszYsXARAEYcYADW2gDKmMxgqFD2yDCsYjUgeBBg5ZXVEV3VrDCpHPZ8Y1WDAFaxAB4AQg3NMC3MtgAyXXBLE5FiMJiXnmeD7azTyCC1zLWcuFBFmKxdMADqBRgAAtkAAOAAMER+EGq-w4mGePzg0xypwiWgAjmJUFo5swFsw7hFquhUL1+oNYl17Ng4ABFMQQeDEkAwN6AvAg5DeNkcVDYGkAYQg6EWKCg0HRIDE1QAKm8shVliYTEA

**Comment by mgrip** [comment]
Fixed in #368
